docs/options.md

```diff
@@ -25,6 +25,13 @@ it would break the behavior of the Operator.
     workaround would be [creating MySQL users](users.md) with `REQUIRE SSL`
     option.
 
+If you provide custom configuration to the Operator with several different ways
+at once, it will choose the only one. First, it looks for a Secret object. If
```

Comment:
`it will choose the only one` - I'm not sure that we need `the` here, to me it would sound better `it will choose only one` (since it is not a specific one, but it depends).
But I am terrible at english so this is just my oppinion. :)